doc/release-notes.md

```diff
@@ -1,9 +1,7 @@
-# Dash Core version v22.0.0
+# Dash Core version v22.1.0
 
-This is a new major version release, bringing new features, various bugfixes
-and other improvements.
-
-This release is **mandatory** for all nodes, as it includes a hard fork.
+This is a new minor version release, bringing new features, various bugfixes
```

Comment:
This isn't a complete sentence. Maybe deleted a line you didn't intend to?

Reply:
How about?
```suggestion
This is a new minor version release, bringing new features, and various bugfixes.
```

Reply:
Sure